### ISSUE TITLE
Train on the CUFED5 

### ISSUE THREAD

**Issue by Lkinyuu** [Tengfei-Wang/DCSR#14]
May I ask for the settings that train on the CUFED5 ? I want to reproduce the same result as in the paper. Like epoch、 learning rate、and so on.

**Comment by jiaxinxie97** [comment]
Hi,
The initial lr is 1e-4, decay at 7-13-20-28-36 epoches. We usually train 20 epoches for a CUFED5 model.

**Comment by ximu1211** [comment]
> The initial lr is 1e-4, decay at 7-13-20-28-36 epoches. We usually train 20 epoches for a CUFED5 model.

Is your description really complete and correct? You mean you usually train 20 epoches for a CUFED5 model，but your learning rate strategy continues to 36 epoches. By the way ，according to the paper, training should have two stages. Thank you!

**Comment by ximu1211** [comment]
@jiaxinxie97  @Tengfei-Wang

**Comment by Tengfei-Wang** [comment]
Hi,
We found the model converged within 15~16 epochs for CUFED5 (with --num_patch=300). So we used the max epoch of 20 in our experiments, though we initially set it to 36.
For the concern about two-stage training, we only employ SRA on CameraFusion dataset, as we wanna obtain 8K images but we do not have ground truth. For CUFED5, the training is one-stage as we can obtain the GT HR images in the datasets.
